docs/self-hosted/kubernetes-maintenance.md

```diff
@@ -22,12 +27,12 @@ Private Packagist and the Replicated Management Console will become temporarily
 curl -sSL https://k8s.kurl.sh/privatepackagistkots | bash -s
 ```
 
-#### Update KOTS for Private Packagist Self-Hosted Kubernetes in an existing cluster
+### Updating Helm installations 
 
-```
-curl https://kots.io/install | bash
-kubectl kots admin-console upgrade -n NAMESPACE
-```
+If you installed Private Packagist Self-Hosted with Helm into your existing cluster, you can update the application with this command:
 
-Replace `NAMESPACE` with the namespace in your cluster where KOTS is installed.
+``` 
+helm upgrade -f values.yaml private-packagist oci://registry.replicated.com/privatepackagistkots/private-packagist --version VERSION
+```
 
+You can find the latest version in the [changelog](changelog). Make sure to compare your existing `values.yaml` file to the [current one](http://packagist.com.lo/docs/self-hosted/kubernetes-helm#annotated-configuration).  
```

Comment:
Needs to clarify that versions can be skipped (don't have to install each version when upgrading e.g. from 1.0.20 to 1.0.24). Probably also that the only way to reliably rollback to a previous version due to a failed deployment is via backup

Reply:
Thanks, ddded this to both kURL and Helm upgrades section then